templates/guide/verify_accuracy.html

```diff
@@ -1,54 +1,9 @@
 {% extends "_base.html" %}
 {% block page_title %}{{ feature.name }} - {% endblock %}
 
-{% block css %}
-<link rel="stylesheet" href="/static/css/forms.css?v={{app_version}}">
-{% endblock %}
-
-{% block subheader %}
-<div id="subheader">
-  <h2 id="breadcrumbs">
-    <a href="/guide/edit/{{ feature_id }}">
-      <iron-icon icon="chromestatus:arrow-back"></iron-icon>
-      Verify feature data for {{ feature.name }}
-    </a>
-  </h2>
-</div>
-{% endblock %}
-
 {% block content %}
-<form name="feature_form" method="POST" action="{{current_path}}">
-  <input type="hidden" name="token" value="{{xsrf_token}}">
```

Comment:
I just realized that there is a defect in this existing template.
We need `<input type="hidden" value="">`
It's OK if value is an empty string.  It is used to avoid clearing out other boolean fields.

Reply:
I see. Should I change this (add `value=""` to the hidden token field) for all the other converted page? I have removed all them because I thought it doesn't matter and the value would be added upon form submission.

Reply:
Correction: it cannot be empty.  It should be nonempty, but the exact value does not matter, such as:
`<input type="hidden" name="form_fields" value="accurate_as_of">`

Reply:
The new feature page does not need form_fields because (a) there are no checkboxes on that form and (b) there could not be any True boolean fields that would be inappropriately cleared since the feature is new.

The flat form fields actually does need form_fields because even though the UI says "edit all fields", there may be some fields that are not included in the flat forms.  Any boolean fields that is not part of the flat forms would be cleared in the feature, which would be a surprise to the user since that checkbox was not even offered to them.

form_fields will definitely be needed on the stage editing form.

Reply:
For the edit_all page, it looks like the existing django template was already missing the form_fields hidden field.  So, you did not introduce a new defect in either that page or this one.  But, this is an opportunity to fix it here for the accuracy page.

Reply:
I see. I have added the hidden name="form_fields" input to the forms in verify_accuracy page and edit_all page. This should fix the defect in both pages.